pages/BrowseBySectionHandler.inc.php

```diff
@@ -80,12 +80,25 @@ public function view($args, $request) {
 			$browseByPerPage = BROWSEBYSECTION_DEFAULT_PER_PAGE;
 		}
 
+		$browseByOrder = $section->getData('browseByOrder');
```

Comment:
For backwards compatibility, when unset we should default this to "datePubDesc".